products/render-instruments.js

```diff
@@ -0,0 +1,34 @@
+function renderInstruments(instrument) {
+    const li = document.createElement('li');
+    //li.setAttribute('id', instrument.id);
+    li.className = instrument.category;
+    //li.setAttribute('name', instrument.name);
```

Comment:
Go ahead and remove unused/commented-out code when you're ready to submit a PR, so it looks professional and tidy :)